src/client/runner/automation/key-press-simulator.js

```diff
@@ -0,0 +1,137 @@
+import hammerhead from '../deps/hammerhead';
```

Comment:
Is `key-press-simulator` used by `PressAutomation` only? If so, maybe we should move it to `/automation/press/` ?
/cc @helen-dikareva